src/braket/jobs/hybrid_job.py

```diff
@@ -170,6 +171,8 @@ def job_wrapper(*args, **kwargs) -> Callable:
             Returns:
                 Callable: the callable for creating a Hybrid Job.
             """
+            inner_source_temp_dir, inner_source_file_path = persist_inner_function_source(entry_point)
```

Comment:
curious, why do you use a separate temp dir for this file? could you just use the `temp_dir` created in the `with` statement below? (side note, the `with tempfile.TemporaryDirectory()` pattern automatically deletes the dir when leaving the scope, so you wouldn't have to explicitly call `rmtree`)

Reply:
Actually, there is already two context manager in that code block, I think I can just add the third one for `inner_source_file_path`. I'll update it.